app/models/topic.rb

```diff
@@ -835,6 +837,26 @@ def invite_by_email(invited_by, email, group_ids = nil, custom_message = nil)
     Invite.invite_by_email(email, invited_by, self, group_ids, custom_message)
   end
 
+  def invite_existing_muted?(target_user, invited_by)
+    if invited_by.id &&
+       MutedUser.where(user_id: target_user.id, muted_user_id: invited_by.id)
+           .joins(:muted_user)
```

Comment:
hmm 4 spaces are use for indentation here.

Reply:
That is because Rubocop wants 4 lines here as per:

> Use 4 (not 2) spaces for indenting a condition in an if statement spanning multiple lines.

Reply:
o wow that is a weird rule.